docs/en/week10/10-1.md

```diff
@@ -0,0 +1,271 @@
+
+3. **Videos:** Frames that are closeby in temporal space are related, and those far-away are unrelated
+   
+4. **Multimodal - Video and Audio:** Video sequences along with their corresponding audio are related. Video sequences from one video and audio from another are unrelated.
+   
+5. **Video Object Tracking:** An object is tracked through multiple frames in a video. Detection patches from the same video are related. Detection patches from different videos are unrelated.
+
+
+**What is the fundamental property of Contrastive Learning that prevents trivial solutions?**
+
+<center>
+<div style="text-align: center;"><img src="{{site.baseurl}}/images/week10/10-1/CL_objective.png" style="width: 400px; background-color:#DCDCDC;" /><br></div>
+Figure 6: Loss function of Contrastive Learning
+</center>
+
+The objective function of Contrastive Learning is designed to prevent trivial solutions. The aim of the function is to make sure the distance between positive embedding pairs are smaller than the distance between negative embedding pairs. Hence, a trivial solution, where a constant distance is assigned to all embedding pairs, is impossible to attain through a minimization process on the given objective function. However, research has shown that good negative pairs are crucial for Contrastive Learning to work well.
+
+
+
+
+## SimCLR
+In SimCLR, Simple Framework for Contrastive Learning of Visual Representations, two correlated views of an image are created using augmentations like random cropping, resize, color distortions and Gaussian blur. A base encoder is used along with a projection head to obtain representations that are used to perform contrastive learning. 
+
+It utilizes a large batch size to generate negative samples.  The batch can be spread across multiple GPUs independently. Images from one GPU could be considered as negatives for those from another GPU.
+
+**Advantages of SimCLR:**
+1. Simple to implement
+
+**Disadvantages of SimCLR:**
+1. Large batch size required
+2. Large number of GPUs required
+
+**How do we solve the compute problem in the SimcLR?**
+
+A **Memory bank** can be used to maintain a momentum of activations across all features. With each forward pass of an image, the memory bank features are updated using the new embeddings. These memory bank features can then be used as negatives for contrastive loss. 
+
+#### Advantages of Memory bank:
+- Compute efficient. Requires one Forward Pass to compute the embeddings.
+
+#### Drawbacks of Memory bank:
+- Not online. Features become stale very fast.
+- Requires large amount of GPU RAM.
+- Hard to scale storage to millions of images.
+
+## MoCo
+
+MoCo, or, Momentum Contrast, is a contrastive method that utilizes a memory bank to maintain a momentum of activations. It works by using 2 encoders. The $f_{\theta}$ encoder is the encoder that needs to be learnt. $f_{\theta_{EMA}}$ encoder maintains an exponential moving average of $f_{\theta}$.
```

Comment:
`EMA` should be encapsulated into `\text{}` here and everywhere else. It's test, it's not the product of three variables.